Yoast/YoastSEO.js
# 1942 relevant words example

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* ~~Branched from #1953. This PR contains 2 commits:~~
  - ~~Adding: 0fc1fe51a062dd40f0d282c806d57b6fbf8fa53e~~
  - ~~Cleanup: 6f599dd33c8a02cdc787b00417cf4d2b53e6dd3c~~
* Added relevant words from the browserified example to the webpack example.
* Added `react-table` for the UI.

## Test instructions

This PR can be tested by following these steps:

* Go to `examples/webpack`.
* Install dependencies: `yarn`.
* Run it: `yarn start`.
* Check out the relevant words section (possibly compare it to the browserified).

Requires #1953 
Fixes #1942
